invenio_rdm_records/records/api.py

```diff
@@ -228,6 +229,8 @@ class RDMDraft(CommonFieldsMixin, Draft):
 
     has_draft = HasDraftCheckField()
 
+    status = DraftStatus()
```

Comment:
I think we need a status field on the record as well for the index. It's probably a constant field simply saying ``published``.